CDP4CrossViewEditor/Converters/ReactiveThingToObjectListConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ReactiveThingToObjectListConverter.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Cozmin Velciu, Adrian Chivu
+//
+//    This file is part of CDP4-IME Community Edition.
+//    The CDP4-IME Community Edition is the RHEA Concurrent Design Desktop Application and Excel Integration
+//    compliant with ECSS-E-TM-10-25 Annex A and Annex C.
+//
+//    The CDP4-IME Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or any later version.
+//
+//    The CDP4-IME Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4CrossViewEditor.Converters
+{
+    using CDP4Common.CommonData;
+    using CDP4Composition.Converters;
+    using ReactiveUI;
+
+    /// <summary>
+    /// A converter that convert <see cref="ReactiveList{T}"/> to <see cref="ReactiveList{Thing}"/>
+    /// </summary>
+    public class ReactiveThingToObjectListConverter : GenericListToObjectListConverter<Thing>
```

Comment:
Do we explicitly need this class? if We don't plan to add more functionality to it later I think it might be better to remove it and just use `GenericListToObjectListConverter<Thing>` directly; this would also us to remove the tests for this pass-through class.

Reply:
Agree because it was required by the CheckBox lists and shortly this will be replaced with a grid control.
Also tests will be removed.